docs/python-2-3-compatibility.md

```diff
@@ -5,8 +5,8 @@ compatible with both Python versions 2 and 3. More specifically, the
 following Python versions are currently supported:
 
 - Python 2.6.x (support was removed in EasyBuild v4.4.0)
-- Python 2.7.x
-- Python 3.5.x
+- Python 2.7.x (support will be removed in EasyBuild v5.0.0)
+- Python 3.5.x (support will be removed in EasyBuild v5.0.0)
```

Comment:
link to `easybuild-v5/overview-of-changes.md`

Reply:
that file has `# Overview of changes in EasyBuild version 5.0 {: #overview }`  line in it, where the `overview` tag is quite not specific...should I change it to sth like `eb5_overview` ?

Reply:
Either rename to `eb5_changes_overview` (like in `easybuild-v4/overview-of-changes.md`), or just drop it, a tag on top of the page isn't very useful anyway

Reply:
@deniskristak ping?